access-token-management/src/AccessTokenManagement/DistributedClientCredentialsTokenCache.cs

```diff
@@ -11,19 +12,16 @@ namespace Duende.AccessTokenManagement;
 /// Client access token cache using IDistributedCache
 /// </summary>
 public class DistributedClientCredentialsTokenCache(
-    IDistributedCache cache,
+    [FromKeyedServices(ServiceProviderKeys.DistributedClientCredentialsTokenCache)] IDistributedCache cache,
+    TimeProvider time,
```

Comment:
Is this technically a breaking change?

Reply:
Yes. But actually, it's not really needed for this PR, so I removed it.